netjsonconfig/backends/openwrt/renderers.py

```diff
@@ -158,6 +160,18 @@ def _get_interfaces(self):
                 counter += 1
         return uci_interfaces
 
+    def __multi_ipaddr(self, address_list):
```

Comment:
I cannot understand this method very much.

It's better we start again from scratch.